gcpy/benchmark/modules/run_1yr_tt_benchmark.py

```diff
@@ -254,8 +251,8 @@ def run_benchmark(config, bmk_year_ref, bmk_year_dev):
     # Get subset of month datetimes and seconds per month
     # for only benchmark months
     bmk_mons_ref = all_months_ref[bmk_mon_inds]
-    bmk_mons_gchp_ref = all_months_gchp_ref[bmk_mon_inds]
-    bmk_sec_per_month_ref = sec_per_month_ref[bmk_mon_inds]
+    #bmk_mons_gchp_ref = all_months_gchp_ref[bmk_mon_inds]
+    #bmk_sec_per_month_ref = sec_per_month_ref[bmk_mon_inds]
```

Comment:
Can these lines be removed?

Reply:
@msulprizio: Now fixed in commit 3b380be2.